pages/conditional-includes.md

```diff
@@ -0,0 +1,92 @@
+import { Meta } from '../components'
+
+<Meta
+  title="Keeping up with two git identities"
+  description="You can work with your company projects and with open-source projects without mixing two identities."
+/>
+
+# Keeping up with two git identities
+
+_November 13, 2018_
+
+If you are working in the frontend, then you most likely use a lot of open-source projects and perhaps your company even open sourced several internal projects. Also you most likely fix issues you have with 3rd party projects you depend on from time to time.
+
+And everything works fine with you simple global git name and email, but then you start to notice some of your GitHub commits lack an avatar, or quite the opposite in your internal source control system (e.g. hosted BitBucket or Gitlab) your commits lack the avatar and username link is shaded.
```

Comment:
* Not sure about starting a sentence with "And"
* "with you" -> "with your"

Reply:
fixed